test/interactions/interactionTests.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Interactions", () => {
+  describe("Interactions", () => {
```

Comment:
I actually meant `Interaction` for the second one. Do you mind double nesting with this renaming?